tests/docker/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+    irods:
+        build:
+            context: . 
+            dockerfile: irods.ubuntu20.Dockerfile
+        ports:
+            - "1247"
```

Comment:
I believe so as the globus server has to connect to port 1247 on the irods server. ?

Reply:
Oh, this isn't exposing the port on the host machine. I was under the impression that any Docker containers connected via a Docker network could talk on any port unless you configure it otherwise. Maybe not?

Reply:
You might be right.  I will test it and remove if not needed.

Reply:
The ports wasn't needed.  The tests pass without it.  I also logged onto the globus server and manually ran icommands.

This yml was based on the one for the S3 API.  I think for my own purposes I needed that port available externally,.

I removed the port section and pushed the commit.